reference/architecture/index.md

```diff
@@ -128,8 +128,11 @@ By default Spinnaker binds ports according to the following table
 | Fiat        | 7003              |
 | Front50     | 8080              |
 | Gate        | 8084              |
+| Gate x509   | 8085              |
 | Halyard     | 8064              |
 | Igor        | 8088              |
 | Kayenta     | 8090              |
 | Orca        | 8083              |
 | Rosco       | 8087              |
+| Redis       | 6379              |
```

Comment:
Would it make sense to pull these into their own section, as we're really documenting defaults of external services we don't control here?  It does seem useful to have the ports they listen on handy as you often need to run commands against them to check the state of data, but maybe as an "external services" section?

Reply:
I like this idea!! Will add these edits soon thx